src/panels/lovelace/editor/config-elements/hui-conditional-card-editor.ts

```diff
@@ -0,0 +1,277 @@
+                              "ui.panel.lovelace.editor.card.conditional.state_not_equal"
+                            )}</paper-item
+                          >
+                        </paper-listbox>
+                      </paper-dropdown-menu>
+                      <paper-input
+                        .label="${this.hass!.localize(
+                          "ui.panel.lovelace.editor.card.generic.state"
+                        )} (${this.hass!.localize(
+                          "ui.panel.lovelace.editor.card.conditional.current_state"
+                        )}: '${this.hass?.states[cond.entity].state}')"
+                        .value=${cond.state_not !== undefined
+                          ? cond.state_not
+                          : cond.state}
+                        .index=${idx}
+                        .configValue=${"state"}
+                        @value-changed=${this._changeCondition}
+                      ></paper-input>
+                    </div>
+                  </div>
+                `;
+              })}
+              <div class="condition">
+                <ha-entity-picker
+                  .hass=${this.hass}
+                  @change=${this._addCondition}
+                ></ha-entity-picker>
+              </div>
+            </div>
+          `}
+    `;
+  }
+
+  private _selectTab(ev: Event): void {
+    this._cardTab = parseInt((ev.target! as any).selected!, 10) === 1;
+  }
+
+  private _handleCardChanged(ev: CustomEvent): void {
+    ev.stopPropagation();
+    if (!this._config) {
+      return;
+    }
+    this._config.card = ev.detail.config;
+    fireEvent(this, "config-changed", { config: this._config });
+  }
+  private _handleReplaceCard(): void {
+    if (!this._config) {
+      return;
+    }
+    this._config.card = {};
+    fireEvent(this, "config-changed", { config: this._config });
+  }
+
+  private _addCondition(ev: Event): void {
+    const target = ev.target! as any;
+    if (target.value === "" || !this._config) {
+      return;
+    }
+    this._config.conditions.push({
+      entity: target.value,
+      state: "",
+    });
+    target.value = "";
+    fireEvent(this, "config-changed", { config: this._config });
+  }
+  private _changeCondition(ev: Event): void {
+    const target = ev.target as any;
+    if (!this._config || !target) {
+      return;
+    }
+    if (target.configValue === "entity" && target.value === "") {
+      this._config.conditions.splice(target.index, 1);
+    } else {
+      const condition = this._config.conditions[target.index];
+      if (target.configValue === "entity") {
+        condition.entity = target.value;
+      } else if (target.configValue === "state") {
+        if (condition.state_not !== undefined) {
+          condition.state_not = target.value;
+        } else {
+          condition.state = target.value;
+        }
+      } else if (target.configValue === "invert") {
+        if (parseInt(target.selected, 10) === 1) {
```

Comment:
Is selected a string or why is this needed?
```suggestion
        if (Number(target.selected) === 1) {
```
or
```suggestion
        if (target.selected === "1") {
```

Reply:
You said it was
![Skärmavbild 2020-03-05 kl  22 28 10](https://user-images.githubusercontent.com/1299821/76027261-a33d6480-5f30-11ea-916d-cdfd9a4bff0a.png)

I think it isn't, but better safe than sorry in this case, I think.

Reply:
Never mind, that iron-tabs.

Reply:
I did a Science. It's a number.